django/contrib/admin/widgets.py

```diff
@@ -2,16 +2,18 @@
 Form Widget classes specific to the Django admin site.
 """
 import copy
+import json
 
 from django import forms
 from django.conf import settings
+from django.contrib.staticfiles import finders
```

Comment:
That depends on how you define require. It doesn't throw an error, but it doesn't work ether.
 How do you get your static files?
<img width="544" alt="screen shot 2017-07-22 at 12 00 20" src="https://user-images.githubusercontent.com/1772890/28490242-6dbc2c0e-6ed5-11e7-83d9-677262ab720c.png">
Anyways, I will add an explicit list of supported locales.